TCAT/Controllers/FavoritesTableViewController.swift

```diff
@@ -64,18 +64,20 @@ class FavoritesTableViewController: UITableViewController {
             let desiredViewController = CustomNavigationController(rootViewController: rootVC)
 
             if let appDelegate = UIApplication.shared.delegate as? AppDelegate {
-                let snapshot: UIView = appDelegate.window!.snapshotView(afterScreenUpdates: true)!
-                desiredViewController.view.addSubview(snapshot)
-                
-                appDelegate.window?.rootViewController = desiredViewController
-                userDefaults.setValue(true, forKey: Constants.UserDefaults.onboardingShown)
-
-                UIView.animate(withDuration: 0.5, animations: {
-                    snapshot.layer.opacity = 0
-                    snapshot.layer.transform = CATransform3DMakeScale(1.5, 1.5, 1.5)
-                }, completion: { _ in
-                    snapshot.removeFromSuperview()
-                })
+                if let window = appDelegate.window {
+                    let snapshot: UIView = window.snapshotView(afterScreenUpdates: true)!
```

Comment:
Could go for another if let but idk if this would look too good. Any suggestions on how to unwrap without so many nested if lets?

Reply:
You can put multiple if lets on the same line and use the if-let var created.
E.g.
```
If let window = appDelegate.window, let snapshot = window.snapshotView() {
...
}
```

Reply:
I personally like

```
if
    let variable1 = aaa,
    let variable2 = bbb
{
    // blah blah
}